modules/discoveryBidAdapter.js

```diff
@@ -0,0 +1,525 @@
+import * as utils from '../src/utils.js';
+import { getStorageManager } from '../src/storageManager.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'DiscoveryDSP';
```

Comment:
Please see bidder code rules under the Overview section here: https://docs.prebid.org/dev-docs/bidder-adaptor.html

Codes must be lowercased and really should be under 8 characters as to not be truncated. I would suggest simply using discovery as it is also the name of the adapter